google_fonts/lib/src/README.md

```diff
@@ -0,0 +1,4 @@
+To generate the GoogleFonts class:
+
+1. Navigate to the root directory of this project
```

Comment:
Are there extra steps?

Reply:
Just running the dart script (step 2). That will create the generated output file